src/plugins/.common/structures/DataHandling/DatabaseHandler.js

```diff
@@ -1,13 +1,246 @@
 const quickDB = require("quick.db");
+const { Guild, Channel } = require("./../../../../../api/models/index.js");
 
 class DatabaseHandler {
   constructor() {
+    this.guild = new GuildHandler();
+    this.invite = new InviteHandler();
+    this.channels = new ChannelsHandler();
     this.bean = new BeanHandler();
     this.reminder = new ReminderHandler();
-    this.invite = new InviteHandler();
     this.thread = new ThreadHandler();
     this.currency = new CurrencyHandler();
-    this.newUserRoles = new NewUserRolesHandler();
+  }
+}
+
+class GuildHandler {
+  constructor() {}
+
+  async createGuild(guildId) {
+    console.log(`Creating guild with id: ${guildId}`);
+    if (this.getGuild(guildId, false) !== null) return false;
```

Comment:
Usually IDs are generated by the database, so there won't be conflict. Where is guildId coming from in your case?

Reply:
guildId is for the discordId column. Would it be best for me to change the param name?
The this.getGuild(guildId...) is checking if the guild already exists or not.